### ISSUE TITLE
Detect if input string is RFC1918

### ISSUE THREAD

**Issue by sajal** [turbobytes/pulse#32]
Once turbobytes/geoipdb#8 is implemented, check the error type and return ASN `RFC1918` and AS Name `Local Network` when performing lookups.

Continue sending `nil` like we currently do for all other errors.

**Comment by coolparadox** [comment]
Branch `detect_private_network` implements `RFC1918` and `Local Network` as replacement AS number and name when LookupAsn returns error due to private network IP.

As it requires newly created [geoipdb/iputils](/turbobytes/geoipdb/tree/detect_private_network/iputils) subpackage not yet in master, travis builds are failing at this moment. This should be fixed when [geoipdb#25](/turbobytes/geoipdb/pull/25) is merged.

**Comment by coolparadox** [comment]
Branch `detect_private_network` contains patches to enable the backend to label local IP lookups "_RFC1918 Local Network_" as requested:

```
$ http GET localhost:7778/asnlookup/ip/192.168.0.102

{
    "ip": "192.168.0.102",
    "result": {
        "asndb": {
            "asn": "",
            "err": "ASN not found",
            "name": ""
        },
        "cymru": {
            "asn": "",
            "err": "empty asn parameter",
            "name": ""
        },
        "geoipdb": {
            "asn": "RFC1918",
            "err": "",
            "name": "Local Network"
        },
        "ipinfo": {
            "asn": "",
            "err": "ipinfo.io lookup failed for '192.168.0.102': undefined",
            "name": ""
        },
        "maxmind": {
            "asn": "",
            "err": "",
            "name": ""
        }
    }
}
```

In my dev pulse system with a single local agent:

```
$ http GET localhost:7778/agents/

[
    {
        "ASN": "RFC1918",
        "ASName": "Local Network",
        "City": "Paulinia",
        "Connected": true,
        "ConnectedFor": "3m52.822113768s",
        "Country": "Brazil",
        "FirstOnline": "2016-12-02 13:26:05.414813049 +0000 UTC",
        "Host": "",
        "HostType": "H",
        "LatLng": "",
        "Name": "coolparadox-home",
        "Resolvers": [
            ""
        ],
        "Serial": 361880659,
        "State": "SP"
    }
]
```
